web/html/src/manager/systems/all-list.tsx

```diff
@@ -42,8 +42,13 @@ export function AllSystems(props: Props) {
         >
           <IconTag type="header-help" />
         </a>
-      </h1>
 
+        <div className="pull-right btn-group">
+          <a id="addsystem" title="Add a system" className="btn btn-link" href="/rhn/manager/systems/bootstrap">
+            <i className="fa fa-plus"></i>Add System
```

Comment:
This looks good otherwise, but it needs a translation wrapper: `{t("Add System")}`